.gitattributes

```diff
@@ -0,0 +1,62 @@
+# Set default behavior to automatically normalize line endings.
```

Comment:
Do we really need this? Specially all the 'auto', isn't auto the default?
Should we cover the formats we don't care about e.g. doc/pdf/rtf?
I don't remember where we are towards line endings. @RyanSkraba maybe?

Reply:
There was a discussion on this about AVRO-3150 -- some tests were failing due to an overzealous checkstyle check IIRC.

Is there something that fails if this `.gitattributes` isn't present?   I'm not familiar with all the attributes, I admit!  What does `diff` and `astextplain` do? 

I have no objection to enforcing that script files are CRLF/LF no matter where they're checked out!